app/views/layouts/application.html.slim

```diff
@@ -15,6 +15,11 @@ html
       a href="#{root_path}" class="navbar-brand"
         | My Own Blog
 
+      ul class="navbar-nav"
+        - unless current_page?(pages_about_path)
```

Comment:
I think we should always show a link to this page. Usually the links in the navigation menu don't change. Only the menu item where the user is located is highlighted.

Reply:
I agree. Only it seemed ugly to me that when we switch to "About me", we have almost two identical inscriptions in one place. When new items appear, I plan to do highlighting.